Fix grammar and wording (#6)
reliqarts/laravel-scavenger: readme.md

## Configuration

Scavenger is highly configurable. Once configured, the settings will be used for every scrape.

### Structure

> Below is an example of a typical config file structure, with comments explaining the settings.

Reviewer: ```suggestion
Below is an example of a typical config file structure, with comments explaining each setting.
```

- `dissect`: Split compound attributes into smaller attributes via REGEX. (optional)
- `preprocess`: Array of attributes which need to be preprocessed. `[attributeName => callable]` (optional)
- `remap`: Array of attributes which need to be renamed in order to be saved as target objects. `[attributeName => newName]` (optional)

> - `bad_words`: Any scrapes found containing these words will be discarded. (optional)

Reviewer: ```suggestion
- `bad_words`: Any scrap found containing these words will be discarded. (optional)
```

This is referring to a scrap as defined on line [285](https://github.com/reliqarts/laravel-scavenger/pull/6/files#diff-5a831ea67cf5cf8703b0de46901ab25bd191f56b320053be9332d9a3b0d01d15R285)

Author: See comment above

## Glossary of Terms
The following words may appear in context above.

- `Daemon`: User instance to be used by the Scavenger service.
- `Scrap`: Scraped data before being converted to the target object.
- `Target`: Configured source-model mapping for a single entity. 

> - `Target Object`: Eloquent model object to be generated from scrape. 

Reviewer: ```suggestion
- `Target Object`: Eloquent model object to be generated from scrap. 
```

This is referring to a "scrap", as defined on line [285](https://github.com/reliqarts/laravel-scavenger/pull/6/files#diff-5a831ea67cf5cf8703b0de46901ab25bd191f56b320053be9332d9a3b0d01d15R285)

Author: But "scrap" is "a scrap of paper".  And "scrape" is "I am going to scrape that webpage"

Reviewer: Makes sense, however a "scrap" is defined in this context as:

`Scraped data before being converted to the target object.`

"Scrap" is used consistently throughout the package to have this meaning.

Author: Sure, but that's a failure in naming, and not the fault of the README. It really should be `scrape`. However, I won't push the point any further, and accept your suggestions.
